### PR TITLE
chore(i18n): update locales/ja.json

### PR DESCRIPTION
**Pull request by ma10**
Update for ja.json, to add missing translation as well as correct a few existing translation that are obviously misleading.  (See below for the details.)

    Add ja translation for td-headers-attr.incomplete
    Add ja translation for svg-non-empty-title.{fail,incomplete}
    Correct ja translation for svg-non-empty-title.pass
    Add ja translation for presentational-role
    Add ja translation for non-empty-value.fail
    Correct ja translation for non-empty-value.pass
    Add ja translation for non-empty-title.fail
    Add ja translation for non-empty-placeholder
    Add ja translation for non-empty-alt.fail
    Add ja translation for has-visible-text.incomplete
    Add ja translation for button-has-visible-text.incomplete
    Add ja translation for aria-labelledby.incomplete
    Add ja translation for has-lang.fail
    Add ja translation for implicit-label.incomplete
    Add ja translation for hidden-explicit-label.incomplete
    Add ja translation for explicit-label.incomplete
    Add ja translation for no-focusable-content
    Add ja translation for frame-focusable-content
    Add ja translation for focusable-no-name.incomplete
    Add ja translation for color-contrast.incomplete.pseudoContent
    Add ja translation for is-element-focusable
    Add ja translation for has-global-aria-attribute
    Add ja translation for aria-prohibited-attr
    Add ja translation for aria-errormessage.incomplete
    Add ja translation for select-name
    Add ja translation for presentation-role-conflict
    Add ja translation for nested-interactive
    Add ja translation for frame-focusable-content
    Add ja translation for aria-treeitem-name
    Add ja translation for aria-tooltip-name
    Add ja translation for aria-text
    Add ja translation for aria-progressbar-name
    Add ja translation for aria-meter-name
    Add ja translation for aria-dialog-name
    Add ja translation for aria-command-name
    grunt translate --lang=ja (ran at Thu Feb 18 12:39:25 2021 +0900)

Closes issue:

### CODE REVIEW

**Comment by WilcoFiers** [comment]
@ma10 Thank you for the update! Much appreciated. I'm going to need to get some help reviewing your changes. This may take some time.

**Comment by kiyochika** [comment]
@WilcoFiers Sorry for taking a long time on this. This is Kiyochika from Mitsue-Links, and I will review on this PR. As I told you and Steven in e-mail, I am going to comment in Japanese on the translation.

@ma10 Sorry for taking a long time. I will either make suggestions, comment on translated strings, or ask questions.

File: locales/ja.json
Comment (kiyochika) [inline locales/ja.json:865]:
```suggestion
        "noAttr": "要素にvalue属性が指定されていません",
        "emptyAttr": "要素に空のvalue属性が指定されています"
```
「には」→「に」として他とあわせます。

File: locales/ja.json
Comment (kiyochika) [inline locales/ja.json:871]:
```suggestion
        "default": "要素のデフォルトのセマンティクスはrole=\"none\"またはrole=\"presentation\"で上書きされませんでした",
```
スペース削除で統一しました。

File: locales/ja.json
Comment (kiyochika) [inline locales/ja.json:874]:
```suggestion
        "globalAria": "要素にはARIAのグローバル属性が指定されているため、roleはpresentation系のものになりません",
        "focusable": "要素がフォーカス可能なため、roleはpresentation系のものになりません",
        "both": "要素にARIAのグローバル属性が指定されておりフォーカス可能なため、roleはpresentation系のものになりません"
```
「要素」の変換漏れをひとつ修正しました。
「presentation系のもの」という訳は修正していないのですが少し違和感がありました。既存の訳ではないと思うので、少し検討させてください。

**Review by kiyochika — CHANGES_REQUESTED** [review]
@ma10 文体の調整が主な内容ですが、修正希望箇所に変更提案を作成しました。一部コメントがありますので、その部分には返信いただければと思います。

File: locales/ja.json
Comment (ma10) [reply locales/ja.json:873]: 初めて見ると伝わりづらい訳ですね。
ただチェックの内容を考えると他にあまり良い案もなく。
意訳して、 "none" または "presentation" ではない、とするのも1つの方法かもしれません。

Comment (kiyochika) [reply locales/ja.json:873]:
```suggestion
        "globalAria": "要素にはARIAのグローバル属性が指定されているため、role=\"none\"またはrole=\"presentation\"にはなりません",
        "focusable": "要素がフォーカス可能なため、role=\"none\"またはrole=\"presentation\"にはなりません",
        "both": "要素にはARIAのグローバル属性が指定されており、フォーカス可能なため、role=\"none\"またはrole=\"presentation\"にはなりません"
```
確認ありがとうございます。そうですね、ということで、修正案を修正しましたので再度ご確認いただけますでしょうか。

**Comment by ma10** [comment]
@WilcoFiers  After getting review from @kiyochika , we've resolved all issues.  There are conflicts with the develop branch, but they are caused, I believe, by ja.json on the develop branch being outdated.

I found a few new messages in the latest message catalog, but I'll work on them after this PR is merged.

**Comment by straker** [comment]
@ma10 Thanks for working with the reviewer to update the translations. I went ahead and merged develop for you and just accepted your changes. Looking at the diff it was either due to translations missing from the develop version for new rules (`aria-text`, `aria-meter-name`, etc.) or due to new fail/incomplete messages that weren't there before.

**Review by straker — APPROVED** [review]

**Comment by straker** [comment]
Reviewed for security.

**Pull request merged**
